<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="nextNode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevNode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initNode">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>init</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="status">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>191</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newNode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>newCurr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeNode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="firstInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="firstResult">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="secondResult">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="newOperation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>swap</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="swapNode">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cacheSaveButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>cachesave</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>250</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>введите операцию</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>введите операцию</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>начальный ввод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>текущая вершина</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newNextNode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>newNext</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>newStart</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newType">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>new type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>openLog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openGraph">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>openGraph</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
